libs/designsystem/src/lib/components/slide-button/slide-button.component.scss

```diff
@@ -2,7 +2,7 @@
 @use 'slide-button.component.shared';
 
 @mixin _slider-thumb() {
-  -webkit-appearance: none;
+  appearance: none;
```

Comment:
I just found out, that apparently [Angular uses Autoprefixer](https://angular.io/guide/build#configuring-browser-compatibility) as part of the build, so letting stylelint remove vendor specific prefixes _should_ be safe in the Sass source files - as long as they get changed to standard properties.

✅ `-webkit-appearance` ➡️ `appearance` is ok (I think)
❌ `display: -webkit-box` ➡️ `display: box` is _not_ ok 

Any thoughts?